lib/app/navigation/router/page_transitions.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:animations/animations.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_template/util/extensions/context_extension.dart';
+import 'package:go_router/go_router.dart';
+
+abstract class PageTransitions {
```

Comment:
Loooove this!!! Isn't `openContainer` transition supported? Thats the coolest one imo